rafthttp/pipeline.go

```diff
@@ -121,6 +121,7 @@ func (p *pipeline) handle() {
 			if isMsgSnap(m) {
 				p.r.ReportSnapshot(m.To, raft.SnapshotFinish)
 			}
+			reportSendingDuration(pipelineMsg, m, time.Since(start))
```

Comment:
we actually do not care about the nano level latency. I am also going to remove the `end` when i start to remove the stats stuff.